### ISSUE TITLE
Unable to use slider controls in iOS with Voiceover

### ISSUE THREAD

**Issue by gmjellin** [Stanko/aria-progress-range-slider#2]
The slider controls are accessible with touch when VoiceOver is off, but when turned on user is unable to move the control. The VO instruction is "swipe up or down with one finger to adjust the value", but when doing so nothing happens. 

Thanks for building this!
Greg

**Comment by Stanko** [comment]
Hello @gmjellin,
Any ideas what is causing it, and how to fix it? I'll take a look, but it may take some time, as I have a lot of going on at the moment.

Cheers!

**Comment by gmjellin** [comment]
Sorry for the delayed response. It appears that neither VO nor TalkBack offer good support for role=slider. I don't think there is much that can be done. If I come across a work around I'll let you know.
